receiver/vcenterreceiver/metadata.yaml

```diff
@@ -201,6 +207,28 @@ metrics:
       value_type: int
       aggregation_temporality: cumulative
     attributes: []
+  vcenter.host.cpu.capacity:
+    enabled: false
+    description: Total CPU capacity of the host system.
+    unit: "MHz"
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation_temporality: cumulative
+    attributes: []
+    warnings:
+      if_enabled_not_set: "this metric will be enabled by default starting in release v0.105.0"
+  vcenter.host.cpu.reserve.capacity:
+    enabled: false
+    description: Total CPU capacity that is available for reserve or reserved by virtual machines.
```

Comment:
I think the description and name of this one has some room for improvement (still sounds a bit confusing).

What do you think about something like `vcenter.host.cpu.reserved` for the name? I know the performance metric names have "capacity" in them, but the UI equivalents do not. "Capacity" seems to only make sense for "total" and not "used" to me.

Then the description could then be something like `The CPU of the host reserved for use by virtual machines.`

Attribute could be changed to `cpu_reservation_type` with values of `total` and `used`. Description could be `The type of CPU reservation for the host`.

These are all just suggestions, but what do you think @BominRahmani ?

Reply:
I was actually teetering back and forth between `vcenter.host.cpu.reserved` and `vcenter.host.cpu.reserve.capacity` I only decided the latter since it felt a bit more verbose. I was also thinking about switching the `cpu_reservation_type` to `total` and `used` to match the UI a bit more originally, So I am totally ok with these changes.